test/parallel/test-fs-read.js

```diff
@@ -79,7 +79,11 @@ assert.throws(
 
 assert.throws(
   () => fs.read(fd, { buffer: null }, common.mustNotCall()),
-  /TypeError: Cannot read properties of null \(reading 'byteLength'\)/,
+  {
+    name: 'TypeError',
+    message: 'The "buffer" argument must be an instance of Buffer, ' +
+    'TypedArray, or DataView. Received null',
+  },
```

Comment:
Let's not validate the message, but rather the error code.
```suggestion
  { code: 'ERR_INVALID_ARG_TYPE' },
```

Reply:
:+1: fixed https://github.com/nodejs/node/pull/44769/commits/b7ba2c36b96c8102115bb86d812459b9ccb96f1d